CHANGELOG.md

```diff
@@ -12,6 +12,62 @@ Types of changes:
 - *Fixed* for any bug fixes.
 - *Security* in case of vulnerabilities.
 
+## 4.12.0
+
+### Fixed
+
+* feat: add SPDX headers by @luka-nextcloud in [#1360](https://github.com/nextcloud/notes/pull/1360)
+* fix: Adjust note item size to be one line by @juliusknorr in [#1387](https://github.com/nextcloud/notes/pull/1387)
+* feat: migrate to files:node:updated by @luka-nextcloud in [#1377](https://github.com/nextcloud/notes/pull/1377)
+* Move to more standard CI pipelines by @juliusknorr in [#1389](https://github.com/nextcloud/notes/pull/1389)
+* Adjust the welcome message paddings for mobile view by @falghamdi125 in [#1425](https://github.com/nextcloud/notes/pull/1425)
+* Migrate notes search to NcTextField with clear button by @korelstar in [#1410](https://github.com/nextcloud/notes/pull/1410)
+* Fix npm audit by @nextcloud-command in [#1415](https://github.com/nextcloud/notes/pull/1415)
+* Micro text change: Remove reference to https://github.com/djmoch/nncli by @per-oestergaard in [#1423](https://github.com/nextcloud/notes/pull/1423)
+* Fix note caption colour for custom themes by @looowizz in [#1460](https://github.com/nextcloud/notes/pull/1460)
+* add notes path to capabilities by @tobiasKaminsky in [#1468](https://github.com/nextcloud/notes/pull/1468)
```

Comment:
Some are features or new additions not fixes, some are rather ci related changes and npm audit would also fit under updating dependencies. Can you clean that up a bit more?